infra/docker-swarm/ansible/roles/testground-services/tasks/main.yaml

```diff
@@ -0,0 +1,53 @@
+- name: Create Testground overlay control network
+  docker_network:
+    name: control
+    driver: overlay
+    attachable: yes
+    enable_ipv6: no
+    ipam_config:
+      - subnet: 10.1.0.0/16
+
+- name: Pull ipfs/testground image
```

Comment:
What is this pull for?

Reply:
Same as comment above - confused, because I thought we need it for `sidecar`. Removed.